actors/stage/src/main/java/com/ea/orbit/actors/cluster/JGroupsClusterPeer.java

```diff
@@ -144,13 +145,21 @@ public Address call()
                         udp.setMulticastPort(udp.getMulticastPort() + ((clusterName.hashCode() & 0x8fff_ffff) % portRangeLength));
                     }
 
+
+                    ProtocolStack stack = baseChannel.getProtocolStack();
+                    FORK fork = (FORK) stack.findProtocol(FORK.class);
+                    if (fork == null)
+                    {
+                        stack.insertProtocol(fork = new FORK(), ProtocolStack.ABOVE, FRAG2.class);
```

Comment:
This shouldn't even be necessary, ForkChannel's forth argument create_fork_if_absent is set to true which does exactly the same thing inside the constructor.. https://github.com/belaban/JGroups/blob/master/src/org/jgroups/fork/ForkChannel.java